lib/layout/Layout.d.ts

```diff
@@ -14,7 +14,7 @@ export interface LayoutProps {
     header?: HeaderConfig;
     menu?: React.ReactElement<MenuItemProps>[];
     structure?: {
-        [key: string]: StructureItem;
+        [key: string]: React.ReactElement<StructureItemProps>;
```

Comment:
@KostNazvanov could we use array of structure items instead of object? Also the StructureItem key property could be used instead of key of current object (as path)